client/local.env

```diff
@@ -0,0 +1,3 @@
+HOSTNAME=localhost
+PORT=8080
+HOST=http://$HOSTNAME:$PORT
```

Comment:
I'd collapsed all these down into just a single `BASE_URL` because port is almost never present in http requests, it only exists in dev because the dev server doesnt use port 80